### PR TITLE
feat: support new pricing model

### PR DESCRIPTION
**Pull request by natapg**
Closes [WEB-299](https://fpjs.myjetbrains.com/youtrack/issue/WEB-299)

It will be merged on January 1, not before

### CODE REVIEW

File: src/helpers/pricing.ts [hunk 1]
```diff
@@ -35,5 +35,5 @@ export function calculatePrice(identifications: number): string {
     notation: 'standard',
   }
 
-  return numberFormatter(currencyFormatOptions).format(identifications / 1000)
+  return numberFormatter(currencyFormatOptions).format(identifications / 500)
```
Comment (Lisenish) [inline src/helpers/pricing.ts:38]: Our new price is 0.02$ per 1000 calls, so can we multiple it by `0.02` instead? So it matches with how we're presenting our pricing.
And also move `0.02` to a constant with a clear name.

**Review by Lisenish — APPROVED** [review]
@natapg Pease make sure that this branch doesn't have conflicts on Friday

**Review by Lisenish — APPROVED** [review]
@natapg Pease make sure that this branch doesn't have conflicts on Friday

**Review by Lisenish — APPROVED** [review]
@natapg Pease make sure that this branch doesn't have conflicts on Friday

**Review by Lisenish — APPROVED** [review]
@natapg Pease make sure that this branch doesn't have conflicts on Friday

**Review by Lisenish — APPROVED** [review]
@natapg Pease make sure that this branch doesn't have conflicts on Friday

File: src/helpers/pricing.ts (hunk 1 above)
Comment (natapg) [reply src/helpers/pricing.ts:38]: good idea

Comment (natapg) [reply src/helpers/pricing.ts:38]: actually, is 0.002$, right? @Lisenish

Comment (Lisenish) [reply src/helpers/pricing.ts]: @natapg Yeah sure, sorry :)

**Pull request merged**